### PR TITLE
Javascript/javascript2/week1

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: javascript/javascript2/week1/findTheShortestWord.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    Find and count the Danish letters function
+
+*/
+
+// Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function wordFinder(list) {
+    let shortestWord = list[0];
```
Comment (cameronjdavis) [inline javascript/javascript2/week1/findTheShortestWord.js:14]: The function `wordFinder()` assumes that `list` will have at least one element. That is true in this example but is a case we should think about.

File: javascript/javascript2/week1/hyfBay/main.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+        productLi.appendChild(header);
+        productLi.appendChild(productPrice);
+        productLi.appendChild(productRating);
+
+        productsUl.appendChild(productLi);
+    });
+
+    let Body = document.querySelector("body");
+    Body.appendChild(productsUl);
```
Comment (cameronjdavis) [inline javascript/javascript2/week1/hyfBay/main.js]: A variable with a capital letter. 😮  I guess that's OK but be consistent when naming variables. All your other variables are lowercase, e.g. `header`.
```suggestion
    let body = document.querySelector("body");
    body.appendChild(productsUl);
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]
I love the design.

From Cameron - Strong Bear

File: javascript/javascript2/week1/hyfBay/main.js (hunk 2 above)
Comment (Hani-far) [reply javascript/javascript2/week1/hyfBay/main.js]: True. Thanks

File: javascript/javascript2/week1/findTheShortestWord.js (hunk 1 above)
Comment (Hani-far) [reply javascript/javascript2/week1/findTheShortestWord.js:14]: Yes. got it. Thanks for the comment.

**Pull request merged**